lib/matplotlib/__init__.py

```diff
@@ -1036,22 +1136,22 @@ def rc_params_from_file(fname, fail_on_error=False, use_default_template=True):
                                       if key not in _all_deprecated])
     config.update(config_from_file)
 
-    verbose.set_level(config['verbose.level'])
-    verbose.set_fileo(config['verbose.fileo'])
+    _set_logger_verbose_level(config['verbose.level'],
+                              config['verbose.fileo'])
 
     if config['datapath'] is None:
         config['datapath'] = get_data_path()
 
+    log = logging.getLogger(__name__)
     if "".join(config['text.latex.preamble']):
-        verbose.report("""
+        log.info("""
```

Comment:
The standard way is to do
```
log.info("format_string %s", args)
```
instead of
```
log.info("format string %s" % args)
```
Note that the latter form will raise an exception if args has the bad idea of including a `%` (see comment below for more details).

Reply:
Additional point of information: the reason the logging module uses a format string followed by arguments is that this means the string interpolation is done only if the logging level is such that the result will actually be used.  In other words, it is a performance optimization.